roles/setup_pemserver/defaults/main.yml

```diff
@@ -19,6 +19,8 @@ pem_agent_service: pemagent
 
 pem_server_bin_path: "/usr/edb/pem/bin"
 pem_agent_bin_path: "/usr/edb/pem/agent/bin"
+pem_server_config_file: "{{ '/etc/httpd/conf.d/edb-ssl-pem.conf' if ansible_os_family == 'RedHat' else '/etc/apache2/sites-available/edb-ssl-pem.conf' }}"
+pem_server_port: "8443"
```

Comment:
What's going on if we change it to `8444`? Is the Apache configuration updated accordingly?

Reply:
I think this should allow the changes
https://github.com/EnterpriseDB/edb-ansible/blob/8901e10c34926578b68053f51c5b5721980837ca/roles/setup_pemserver/tasks/pem_server_config.yml#L25

Reply:
I missed it, sorry.